ote/src/cljs/ote/app/controller/transit_visualization.cljs

```diff
@@ -19,7 +19,7 @@
   (or (:route-hash-id route) (str (:route-short-name route) "-" (:route-long-name route) "-" (:trip-headsign route))))
 
 (def hash-colors
-  ["#E1F4FD" "#DDF1D2" "#FFF7CE" "#E0B6F3" "#A4C9EB" "#FBDEC4" "#FFAD9A" "#BDECB7" "#BAA2D1" "#F89DC0" "#97D5E8" "#E1C7B0" "#9CCDDC" "#c82565"]
+  ["#E1F4FD" "#DDF1D2" "#FFF7CE" "#E0B6F3" "#A4C9EB" "#FBDEC4" "#FFAD9A" "#BDECB7" "#BAA2D1" "#F89DC0" "#97D5E8" "#E1C7B0" "#9CCDDC" "#c56c8f" "#b9cf84" "#a4515b" "#23dbe1"]
```

Comment:
In the future would be good to have def's for these in stye file with a semantic name. `hash-colors` vector should then refer to those symbols.

Reply:
We need correct colors from Jussi. Eight of these are my selections and there are not good.
Would be appropriate to move these to correct place when we get correct colors.